obs-studio-client/source/scene.cpp

```diff
@@ -74,6 +74,7 @@ Napi::Object osn::Scene::Init(Napi::Env env, Napi::Object exports) {
 			InstanceMethod("sendMouseWheel", &osn::Scene::CallSendMouseWheel),
 			InstanceMethod("sendFocus", &osn::Scene::CallSendFocus),
 			InstanceMethod("sendKeyClick", &osn::Scene::CallSendKeyClick),
+			InstanceMethod("call_handler", &osn::Scene::CallCallHandler),
```

Comment:
a name of new method in a different style from other methods. Is it intentional?

Reply:
A mistake on my part - had the backend style in my mind while I was writing this.